QueryBuilder.Tests/QueryBuilderTest.cs

```diff
@@ -71,6 +77,8 @@ public void Offset()
             Assert.Equal("SELECT * FROM (SELECT *, ROW_NUMBER() OVER (ORDER BY (SELECT 0)) AS [row_num] FROM [users]) AS [subquery] WHERE [row_num] >= 11", c[0]);
             Assert.Equal("SELECT * FROM `users` LIMIT 18446744073709551615 OFFSET 10", c[1]);
             Assert.Equal("SELECT * FROM \"users\" OFFSET 10", c[2]);
+            Assert.Equal("SELECT * FROM [users] OFFSET 10", c[3]);
```

Comment:
This syntax is not supported on sqlite https://www.sqlite.org/syntax/simple-select-stmt.html

Reply:
@pm100 

**A hint:** 

Method `CompileOffset`

**Link:** https://github.com/sqlkata/querybuilder/blob/master/QueryBuilder/Compilers/MySqlCompiler.cs

To solve the problem of "offset"